src/mods.cpp

```diff
@@ -60,28 +59,55 @@ void parseModContents(ModSpec &spec)
 
 	// Handle modpacks (defined by containing modpack.txt)
 	std::ifstream modpack_is((spec.path+DIR_DELIM+"modpack.txt").c_str());
-	if(modpack_is.good()){ //a modpack, recursively get the mods in it
+	if (modpack_is.good()) { // a modpack, recursively get the mods in it
 		modpack_is.close(); // We don't actually need the file
 		spec.is_modpack = true;
 		spec.modpack_content = getModsInPath(spec.path, true);
-
 		// modpacks have no dependencies; they are defined and
 		// tracked separately for each mod in the modpack
-	}
-	else{ // not a modpack, parse the dependencies
-		std::ifstream is((spec.path+DIR_DELIM+"depends.txt").c_str());
-		while(is.good()){
-			std::string dep;
-			std::set<char> symbols;
-			if(parseDependsLine(is, dep, symbols)){
-				if(symbols.count('?') != 0){
-					spec.optdepends.insert(dep);
-				}
-				else{
-					spec.depends.insert(dep);
+
+	} else {
+		// Attempt to load dependencies from mod.conf
```

Comment:
The alternative is to make a function and expose it to Lua. I'll have a look and see if if there's a good place for this. If there isn't, then for such small code it's not worth refactoring imo.